R/waterfall_build_proportions.R

```diff
@@ -39,9 +39,9 @@ waterfall_build_proportions <- function(data_frame, plot_palette,
         theme(
             axis.ticks.x = element_blank(),
             axis.text.x =  element_blank(),
-            axis.title.x =if (x_label) element_text() else element_blank(),
+            axis.title.x = if (x_label) element_text() else element_blank(),
```

Comment:
Moving the gene labels from the main heatmap/mutational landscape to the gene plot means that the axis titles for the mutational burden and clinical/proportion plots will be closer to their axes